examples/emoji-color-cairo.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
```

Comment:
That came from the old one (the old emoji-color). I 'll change it to plain python now. I like to shebang if it can run as a script (not all python code does), rather than intended for import.